python/cuml/test/dask/test_kmeans.py

```diff
@@ -193,4 +193,4 @@ def test_score(nrows, ncols, nclusters, n_parts,
     local_model = cumlModel.get_combined_model()
     expected_score = local_model.score(X_train.compute())
 
-    assert actual_score == expected_score
+    assert abs(actual_score - expected_score) < 1e-3
```

Comment:
Why was 1e-3 in particular chosen?

Reply:
Because it's small enough compared to the numbers in the assertion (15k and 5k).

The error reported in the issue were in the order of 2e-4.